### PR TITLE
adding spicedb deployment spec

### PR DESCRIPTION
**Pull request by akoserwal**
- Adding basic single pod spicedb deployment k8s template

### CODE REVIEW

File: k8s/spicedb.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            - name: "prometheus"
+              containerPort: 9090
+              protocol: "TCP"
+          readinessProbe:
+            exec:
+              command: ["grpc_health_probe", "-v", "-addr=localhost:50051"]
+            failureThreshold: 5
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 5
```
Comment (Rajagopalan-Ranganathan) [inline k8s/spicedb.yaml:90]: a new line at the end of the file please

File: k8s/spicedb.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+# It runs with the following:
+#   - a single node deployment
+#   - default ports (gRPC 50051, dashboard 8080)
+#   - no TLS
+#   - debug logging
+#   - in-memory datastore
+#
+# To apply this configuration execute the following:
+# kubectl -n $YOUR_NAMESPACE create secret generic spicedb --from-literal=SPICEDB_GRPC_PRESHARED_KEY=$YOUR_SECRET
+# kubectl -n $YOUR_NAMESPACE apply -f basic.yaml
```
Comment (Rajagopalan-Ranganathan) [inline k8s/spicedb.yaml:12]: should be -f spicedb.yaml ?

File: k8s/spicedb.yaml (hunk 1 above)
Comment (akoserwal) [reply k8s/spicedb.yaml]: added

File: k8s/spicedb.yaml (hunk 2 above)
Comment (akoserwal) [reply k8s/spicedb.yaml]: fixed

**Comment by merlante** [comment]
lgtm

**Pull request merged**